docs/installation/upgrade.md

```diff
@@ -220,6 +220,16 @@ This article describes how to proceed in order to convert your Citrix XenServer
 * Eject CDs from your VMs before upgrading [to avoid issues](https://xcp-ng.org/forum/topic/174/upgrade-from-xenserver-7-1-did-not-work): `xe vm-cd-eject --multiple`
 * **It is very important to make sure clustering is not enabled on your pool**. It's a functionality that relies on proprietary software and that is not available in XCP-ng, and having it enabled before the upgrade will lead to XAPI being unable to start due to unexpected data in the database. If it is enabled or you already upgraded, see [this comment](https://github.com/xcp-ng/xcp/issues/94#issuecomment-437838544).
 * If you already have an XCP-ng pool, do not try to add a slave running XenServer to it. You will get an "Incompatible License" error. Please upgrade the slave to XCP-ng first, then add it to your existing pool
+* Before proceeding, it is important to ensure that the `xapi` service
+  currently running on your host is compatible with the version provided by
+  XCP-ng. To verify this, open a terminal session on the host machine and execute
+  the command `rpm -qi xapi-core`. This command will display detailed information
+  about the installed `xapi-core` package, including its version number. Check the
+  xapi version available in [XCP-ng updates](https://mirrors.xcp-ng.org/8/8.3/updates/Source/SPackages/).
```

Comment:
This could be a check to do in the installer.  It will not be available in some cases (no-network, airgap), and we could warn the user that there is a check to be done manually.

Reply:
card created internally